help/tags/extensions/client/target-v2/release-notes.md

```diff
@@ -9,6 +9,10 @@ exl-id: c1a04e62-026d-4b16-aa70-bc6d5dbe6b2d
 >
 >Adobe Experience Platform Launch has been rebranded as a suite of data collection technologies in Adobe Experience Platform. Several terminology changes have rolled out across the product documentation as a result. Please refer to the following [document](../../../term-updates.md) for a consolidated reference of the terminology changes.
 
+## v0.19.2 (February 14, 2023)
+
+- Bug fix: allow timeout to be set to a data element
```

Comment:
Can we update this to: 

"Fixed issues with allowing timeout to be set to a data element."

Reply:
Thank you for the feedback @hbhandal80 !  I have made the change.